### PR TITLE
Improve docs on FLAME usage

### PR DESCRIPTION
**Pull request by josevalim**

### CODE REVIEW

**Comment by josevalim** [comment]
Let's merge this after #3. :)

File: assets/runner_cell/src/App.js [hunk 1]
```diff
@@ -238,7 +238,13 @@ function HelpBox(_props) {
         .
       </p>
       <p>
-        Once a pool is started, you can execute code on a separate machine as
+        On start, FLAME will package all of your notebook's dependencies and
+        automatically synchronize modules defined within Livebook with remote
+        nodes. Note process state and application configuration are not
```
Comment (jonatanklosko) [inline assets/runner_cell/src/App.js]:
```suggestion
        nodes. Note that process state and application configuration are not
```

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Review by hugobarauna — COMMENTED** [review]
Cut “it will” ?

**Comment by jonatanklosko** [comment]
Fixed!

**Pull request merged**